src/Nethermind/Nethermind.JsonRpc/Modules/Eth/EthModule.cs

```diff
@@ -297,7 +297,20 @@ public ResultWrapper<Keccak> eth_sendRawTransaction(byte[] transaction)
         public ResultWrapper<byte[]> eth_call(TransactionForRpc transactionCall, BlockParameter blockParameter = null)
         {
             BlockHeader block = blockParameter == null ? _blockchainBridge.Head : _blockchainBridge.GetBlock(blockParameter).Header;
-            BlockchainBridge.CallOutput result = _blockchainBridge.Call(block, transactionCall.ToTransaction());
+
+            var tx = transactionCall.ToTransaction();
+            tx.GasPrice = 0;
+            if (tx.GasLimit < 21000)
```

Comment:
From docs: gas: QUANTITY - (optional) Integer of the gas provided for the transaction execution. eth_call 

So maybe lets just mark it as system transaction? Might be security issue though.